lib/aarch32/misc_helpers.S

```diff
@@ -7,6 +7,7 @@
 #include <arch.h>
 #include <asm_macros.S>
 #include <assert_macros.S>
+#include <platform_def.h>
```

Comment:
I disagree with the solution you've chosen. This file should not require anything platform specific. To avoid having compiler specific directives in the source code, I think it would be better to modify the compiler options in armv7-a-cpus.mk. It looks like you can alter the `-mcpu` and `-march` options with `+sec`.

Reply:
I'll check that. If it can be done that way I'll make the change in the next PR.